login.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Log In-MockUp</title>
+		<link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+		<link rel="stylesheet" type="text/css" href="css/login.css">
+	</head>
+	<body>
+	<!-- First section --> 
+		<section>
+			<div id="first-div" class="rectangle firstRectangle">
+				<div id="white-rectangle" class="whiteRectangle"></div>
+				<div id="login-box" class="logInBox">
+					<div id="head-logIn" class="headLogIn"><div>Interview FeedBack</div></div>
+					<div id="content-login" class="contentLogIn">
+						<form id="log-form" class="logFormClass" action="evaluations.html">
+						  <input class="logFormInput" type="text" name="userName" placeholder="Username"><br>
+						  <input class="logFormInput" type="password" name="password" placeholder="Password">
+						  <button class="logFormButton" type="submit value="Submit">Login</button>
+						</form>
+					</div>
+				</div>
+			</div>
+		</section>
+ 	
+ 	<!-- Second section --> 
+		<section>
+			<div id="second-div" class="rectangle secondRectangle">
+				<div id="img-container" class="imgContainer">
+					<img src="img/feed2.jpg">
+				</div>
+				<div id="text-container" class="textContainer">
+					<article>
+						<h2>Submit new feedback!</h2>
+						<div id="tc-body" class="tcBody">
+							Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum
+						</div>
+					</article>
+				</div>
+			</div>
+		</section>
+	
+	<!-- Third section --> 
+		<section>
+			<div id="third-div" class="rectangle thirdRectangle">
+				<div id="text-container" class="textContainer textContainerThird">
+					<article>
```

Comment:
I'm a little bit confused about this "article". I know it's for semantics. I guess there is some benefits for devices used by blind people like "some kind of tool that reads through the page?"